app/assets/scss/application.scss

```diff
@@ -91,42 +91,6 @@ $path: "/suppliers/static/images/";
   }
 }
 
-.isolated-text {
-  margin-bottom: 30px;
-}
-
-.browse-list-borderless {
-  .browse-list-item {
-    border-top: none;
-    padding-top: 0;
-    margin-top: 20px;
-  }
-}
-
-.filter-field-text {
-  @include core-16;
-  @include box-sizing(border-box);
-  width: 100%;
-  padding: 5px;
-}
-
-.return-to-top {
-  display: block;
-  @include core-19;
-  margin: (1.5 * $gutter) 0 (2 * $gutter);
-}
-
 .padding-bottom-small {
   padding-bottom: 10px;
 }
-
-ins {
```

Comment:
So technically I don't think we are using these two (which are used for showing a contract variation) as g cloud 8 is no longer live and is the only framework that has a contract variation at the moment.

However the route still exists to show contract variations, and if we needed one for g10 then we would need to re-add this css. 

https://github.com/alphagov/digitalmarketplace-frameworks/blob/bb4e019354f7308707d1cf5133e19584cf0f1911/frameworks/g-cloud-8/messages/contract_variation_1.yml

Thoughts?

Of note, we also use a 'diffing' style in the admin app but it uses different styling (obviously) but it feels like something that should be moved in to the toolkit at some point in the future.

Reply:
Generic styles on core HTML element types should _definitely_ be in the toolkit, yeah. In light of that, and under the belief that we don't have anything currently using the style, I think we should still opt to remove it for now.

Reply:
Hmm I'm a little torn. Whilst it feels good to remove it (which can then prompt it to be written in the toolkit when we next need a contract variation), it feels a bit wrong to remove css that an html template of ours is relying on and the route still exists (it just happens that it's not in use at the moment).

Reply:
Is a reasonable point. I could go either way.

Probably the latter (keep it) now that you spell it out.

Reply:
I've added a ticket at the bottom of our icebox (https://trello.com/c/N712MbsT/106-move-ins-and-dels-diff-styling-into-the-toolkit). Is it worth adding an inline comment? (feel free to say no, I'm not sure myself).

Reply:
Given adding an inline comment probably takes less time it took you to write this reply, I'd say sure, it can't really not be worth doing. ;)